app/adapters/api-key.js

```diff
@@ -0,0 +1,12 @@
+import ApplicationAdapter from './application';
+
+export default ApplicationAdapter.extend({
+
+    queryRecord() {
+        let url = `${this.buildURL('users', 'me')}token/`;
```

Comment:
It would be good to add a guard here so we can still query api-tokens that aren't the personal token similar to what we do in [the `users` adapter](https://github.com/TryGhost/Ghost-Admin/blob/master/app/adapters/user.js#L6-L8)

Reply:
Thanks for the feedback :pray:  I've updated my code :heavy_check_mark: